packages/lexical-editor/src/components/ToolbarActions/TextAlignmentAction.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import {
+    ElementFormatType,
+    FORMAT_ELEMENT_COMMAND,
+    INDENT_CONTENT_COMMAND,
+    OUTDENT_CONTENT_COMMAND
+} from "lexical";
+import { useRichTextEditor } from "~/hooks/useRichTextEditor";
+import { Compose, makeComposable } from "@webiny/react-composition";
+import { TextAlignmentActionContext } from "~/context/TextAlignmentActionContextProps";
+
+/*
+ * Base composable action component that is mounted on toolbar action as a placeholder for the custom toolbar action.
+ * Note: Toa add custom component access trough @see LexicalEditorConfig API
```

Comment:
Check grammar.

Reply:
I changed the comment a bit, let me know what you think.

```
Base text alignment dropdown composable component.
Note: To add a custom dropdown component use @see LexicalEditorConfig API.
```